.github/workflows/lint.yaml

```diff
@@ -50,3 +50,20 @@ jobs:
         run: python -m pip install gitlint
       - name: Run gitlint check
         run: gitlint --commits origin/${{ github.event.pull_request.base.ref }}..HEAD
+  checkton:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Run Checkton
+        id: checkton
+        uses: chmeliik/checkton@main
```

Comment:
I want to tag this as a version, but the action/repo has no versioning available. We could run into a breakage if the repo is updated.

Reply:
True... In this case I wouldn't make the task mandatory maybe... I would also ask the author (though an issue in their repo) to tag the versions.

Reply:
Created an [Issue](https://github.com/chmeliik/checkton/issues/12)